### PR TITLE
Scale Equivariance

### PR DESCRIPTION
**Pull request by CharlesAuthier**

### CODE REVIEW

File: deep-learning/_posts/2018-08-03-ScaleEquivariance.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+2. The output is composed of the magnitude of the maximally activating scale and of the corresponding scale value itself. These two values are combined into a 2D vector.
+
+3. Convolutions on this vector field use a similarity metric that takes into account both magnitudes and scale factors.
+
+
+## Scale equivariance
+The image name $$x\;\epsilon\;R^{MxN}$$.
+
+A representation $$T$$ of $$G$$ is a function from $$G$$ to a family of linear operators such taht $$g,h\;\epsilon\;G$$: $$T(g)T(h)\;=\;T(gh)$$.
```
Comment (Dref360) [inline deep-learning/_posts/2018-08-03-ScaleEquivariance.md]: that*

File: deep-learning/_posts/2018-08-03-ScaleEquivariance.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Where $$g$$ is the scaling of the object or pattern and $$T^{\chi}(g)$$ to a linear operator thast implements this scaling. $$f$$ is scale *invariant* if: $$f(T^{\chi}(g)x)\;=\;f(x)$$ and *equivariant* if: $$f(T^{\chi}(g)x)\;=\;T^{y}(g)f(x)$$ where $$y$$ is a vector field where scale.
```
Comment (Dref360) [inline deep-learning/_posts/2018-08-03-ScaleEquivariance.md]: that*

File: deep-learning/_posts/2018-08-03-ScaleEquivariance.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ cite:
 This paper shows the effect of the injection of a local scale equivariance into a CNN.
 This highlights the usefulness of allowing for a compact representation that can also learn relationships between different local scales by keeping internal scale equivariance.
 
-The work in three steps:
+The three mains part of this work:
```
Comment (SarahLECLERC7) [inline deep-learning/_posts/2018-08-03-ScaleEquivariance.md]: main *

File: deep-learning/_posts/2018-08-03-ScaleEquivariance.md [hunk 4]
```diff
@@ -26,8 +26,8 @@ The work in three steps:
 ## Scale equivariance
 The image name $$x\;\epsilon\;R^{MxN}$$.
 
-A representation $$T$$ of $$G$$ is a function from $$G$$ to a family of linear operators such taht $$g,h\;\epsilon\;G$$: $$T(g)T(h)\;=\;T(gh)$$.
-Where $$g$$ is the scaling of the object or pattern and $$T^{\chi}(g)$$ to a linear operator thast implements this scaling. $$f$$ is scale *invariant* if: $$f(T^{\chi}(g)x)\;=\;f(x)$$ and *equivariant* if: $$f(T^{\chi}(g)x)\;=\;T^{y}(g)f(x)$$ where $$y$$ is a vector field where scale.
+A representation $$T$$ of $$G$$ is a function from $$G$$ to a family of linear operators such that $$g,h\;\epsilon\;G$$: $$T(g)T(h)\;=\;T(gh)$$.
+Where $$g$$ is the scaling of the object or pattern and $$T^{\chi}(g)$$ to a linear operator that implements this scaling. $$f$$ is scale *invariant* if: $$f(T^{\chi}(g)x)\;=\;f(x)$$ and *equivariant* if: $$f(T^{\chi}(g)x)\;=\;T^{y}(g)f(x)$$ where $$y$$ is a vector field where scale.
```
Comment (SarahLECLERC7) [inline deep-learning/_posts/2018-08-03-ScaleEquivariance.md]: where scale ?

**Pull request merged**